### PR TITLE
Misc queue from 2018 (sorry, lost sources) 

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -6742,6 +6747,11 @@ goin->going
 goind->going
 gonig->going
 gonna->going to, disabled because one might want to allow informal spelling
+gool->ghoul
+gord->gourd
+gormay->gourmet
+gorry->gory
+gost->ghost
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: Russian standard (like ISO, EN, DIN, ASME, AISI, ...)
But maybe too special case and can be ignored manually

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Sorry, I don't understand.

Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: From a quick Google, @arm-in means https://en.wikipedia.org/wiki/GOST . I've not seen it before and I'm aware of most of the others, so probably rather niche.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -8154,9 +8154,13 @@ gigibit->gigabit
 gilotine->guillotine
 gilty->guilty
 ginee->guinea
+gingam->gingham
 gioen->given
 gird->grid, gird,
+girdcols->gridcols
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Specific

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -8154,9 +8154,13 @@ gigibit->gigabit
...
 girds->grids, girds,
+giser->geyser
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Plural

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -8154,9 +8154,13 @@ gigibit->gigabit
...
+gitar->guitar
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Plural

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

**Comment by peternewman** [comment]
I've merged @arm-in . Please shout (or open a PR) if you feel strongly about gost.

**Pull request merged**